src/redux/action/auth.action.jsx

```diff
@@ -0,0 +1,45 @@
+import firebase from "firebase/app";
+import auth from "../../firebase";
+import {
+  LOGIN_REQUEST,
+  LOAD_PROFILE,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+  LOG_OUT,
+} from "../actionType";
```

Comment:
or you can do, import * as ActionTypes. then use it as ActionTypes.abc

Reply:
changed code as per suggestion